src/client/components/BaseLayout/index.jsx

```diff
@@ -53,16 +53,41 @@ const BaseLayout = ({
   const path = useLocation().pathname
   const isIE = useIsIE()
   const message = useSelector((state) => state.user.message)
+
+  // To store y-position to trigger useEffect
+  const prevScrollY = useRef(0)
+  const [isSticky, setIsSticky] = useState(false)
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const currentScrollY = window.scrollY
+      // height of mast
+      if (currentScrollY >= 28) {
+        setIsSticky(true)
+      } else {
+        setIsSticky(false)
+      }
+      prevScrollY.current = currentScrollY
+    }
+
+    window.addEventListener('scroll', handleScroll)
+
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [prevScrollY])
+
   return (
     <>
       <CssBaseline />
       <Masthead />
-      {path === USER_PAGE && isIE && <BannerForIE />}
-      {path === USER_PAGE && message && <Banner text={message} />}
+      {path === USER_PAGE && isIE && <BannerForIE isSticky={isSticky} />}
+      {path === USER_PAGE && message && (
+        <Banner text={message} isSticky={isSticky} />
```

Comment:
@lisatjide I don't think the intention is for the user banner to be sticky, right? I thought the stickiness is just for IE users.

Reply:
Just checked with @lisatjide , the banner should be sticky for both IE and non-IE users